video/out/drm_atomic.h

```diff
@@ -69,6 +70,23 @@ struct drm_object {
     drmModePropertyRes **props_info;
 };
 
+#define DRM_HAS_HDR_METADATA_INFOFRAME
+// https://lists.freedesktop.org/archives/dri-devel/2021-January/295187.html
+#define DRM_HDMI_STATIC_METADATA_TYPE1 0
+
+#include "config.h"
+#if !HAVE_DRM_HDMI_HDR
+#include "osdep/drm-hdr-metadata.h"
+#endif
+
+struct drm_hdr_metadata {
+#ifdef DRM_HAS_HDR_METADATA_INFOFRAME
```

Comment:
This defined is unconditionally set a few lines above. Perhaps you meant to use `HAVE_DRM_HDMI_HDR` through the codebase instead? This way the `drm-hdr-metadata.h` can be removed.